Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundView.swift

```diff
@@ -63,13 +75,20 @@ struct PlaygroundView: View {
 
                     Section(header: Text("Select Use Case")) {
                         VStack(alignment: .leading, spacing: 4) {
-                            Picker("Select Use Case", selection: viewModel.useCase) {
-                                ForEach(PlaygroundConfiguration.UseCase.allCases) {
-                                    Text($0.rawValue.capitalized.replacingOccurrences(of: "_", with: " "))
-                                        .tag($0)
+                            if viewModel.experience.wrappedValue == .instantDebits {
```

Comment:
IMO you could have just not shown this section as users of this likely don't know any different so the extra messaging just adds clutter? I also don't see a "use case" in the [Connections Dev App](https://stripejsdevapp.corp.stripe.com/connections-auth-flow)

Reply:
Updated to hide the use case section when instant debits is selected 👍  

https://github.com/stripe/stripe-ios/assets/172562065/48353c1e-df29-4f00-bc2b-9c07e4e6b4a3